src/components/User/UserHome.vue

```diff
@@ -1,37 +1,59 @@
 <template>
   <v-container>
-    <v-row
-      class="pb-4"
-      no-gutters
+    <v-card
+      class="mb-8 mt-4 grey lighten-4"
+      style="height: 100%;"
     >
-      <h1>
-        View Booking Portal
-      </h1>
-      <v-spacer />
-      <v-btn
-        color="#BDBDBD"
-        class="justify-end"
+      <v-system-bar />
+      <v-toolbar
+        flat
+        height="65%"
       >
-        Sign Out
-      </v-btn>
-    </v-row>
+        <img
+          class="mr-3"
+          src="https://brand.uark.edu/_resources/images/shield.jpg"
+          height="55px"
+          width="55px"
+        >
+
+        <v-toolbar-title
+          class="text-h5"
+        >
+          View Booking Portal
+        </v-toolbar-title>
+        <v-btn
+          depressed
+          :right="true"
+          :absolute="true"
```

Comment:
Similar comment left in UserRes
```suggestion
          right
          absolute
```

Reply:
Done